cves/kernel/CVE-2015-0275.yml

```diff
@@ -1,483 +1,585 @@
+ security bulletins and bug reports. It is not necessarily the same day that
+ the CVE was created.  Leave blank if no date is given.
+
 
-  Please enter your date in YYYY-MM-DD format.
-reported_date:
+ Please enter your date in YYYY-MM-DD format.
+reported_date: '2015-02-18'
 announced_instructions: |
-  Was there a date that this vulnerability was announced to the world? You can
-  find this in changelogs, blogs, bug reports, or perhaps the CVE date.
+ Was there a date that this vulnerability was announced to the world? You can
+ find this in changelogs, blogs, bug reports, or perhaps the CVE date.
 
-  This is not the same as published date in the NVD - that is below.
 
-  Please enter your date in YYYY-MM-DD format.
+ This is not the same as published date in the NVD - that is below.
+
+
+ Please enter your date in YYYY-MM-DD format.
 announced_date: '2015-10-19'
 published_instructions: |
-  Is there a published fix or patch date for this vulnerability?
-  Please enter your date in YYYY-MM-DD format.
-published_date: '2015-10-19'
+ Is there a published fix or patch date for this vulnerability?
+ Please enter your date in YYYY-MM-DD format.
+published_date: '2015-04-03'
 description_instructions: |
-  You can get an initial description from the CVE entry on cve.mitre.org. These
-  descriptions are a fine start, but they can be kind of jargony.
-
-  Rewrite this description IN YOUR OWN WORDS. Make it interesting and easy to
-  read to anyone with some programming experience. We can always pull up the NVD
-  description later to get more technical.
-
-  Try to still be specific in your description, but remove project-specific
-  stuff. Remove references to versions, specific filenames, and other jargon
-  that outsiders to this project would not understand. Technology like "regular
-  expressions" is fine, and security phrases like "invalid write" are fine to
-  keep too.
-
-  Your target audience is people just like you before you took any course in
-  security
-description:
+ You can get an initial description from the CVE entry on cve.mitre.org. These
+ descriptions are a fine start, but they can be kind of jargony.
+
+
+ Rewrite this description IN YOUR OWN WORDS. Make it interesting and easy to
+ read to anyone with some programming experience. We can always pull up the NVD
+ description later to get more technical.
+
+
+ Try to still be specific in your description, but remove project-specific
+ stuff. Remove references to versions, specific filenames, and other jargon
+ that outsiders to this project would not understand. Technology like "regular
+ expressions" is fine, and security phrases like "invalid write" are fine to
+ keep too.
+
+
+ Your target audience is people just like you before you took any course in
+ security
+description: |
+ An issue in the EXT4 Filesystem of the linux kernel caused the system to crash
+ whenever a specific 'fallocate' (used to allocate and manipulate disk space for a file)
```

Comment:
What is the point of using technical jargon if you need to explain it in parathesis? Maybe it's better to completely remove it.